components/hab/src/main.rs

```diff
@@ -1775,10 +1777,9 @@ fn user_param_or_env(m: &ArgMatches<'_>) -> Option<String> {
 // function wouldn't be necessary. In the meantime, though, it'll keep
 // the scope of change contained.
 fn ui() -> UI {
-    let coloring = if hcore::output::is_color() {
-        ColorChoice::Auto
-    } else {
-        ColorChoice::Never
+    let coloring = match output::get_format() {
+        OutputFormat::Color => ColorChoice::Auto,
+        _ => ColorChoice::Never,
```

Comment:
Are we 100% certain that we'll never add any additional output formats? If not, and there's any chance that new format might want to use color, we should be exhaustive instead of using the catch-all:
```suggestion
        OutputFormat::NoColor | OutputFormat::JSON => ColorChoice::Never,
```
That way, the compiler will make sure we update this.

Reply:
I like this suggestion 👍

Reply:
yeah good thinking. This logic is in a couple spots and I'll get it in all of them.